### DISCUSSION TITLE
QField Sync on QGis: All projects disappeared from synchronisation overview

### DISCUSSION THREAD

**Discussion by Notramus** [opengisch/QField#3170] [Q&A]
Hey all,

today after a failed synchronisation all of my QField Sync projects disappeared from the overview. They are still accessible through the online login portal. 

I have tried: 
Reinstalling the plugin
Reinstalling QGis
Logging in and out of the account
Using another computer

What else can I do? The error code seems more confusing than helpfull as well...

![Sample](https://user-images.githubusercontent.com/104778611/183103126-689953cc-24cc-408b-bb53-95afdb85826f.png)

**Comment by Alekssssanderr** [comment]
I have this same problem :/ [{"id":"c0f8877b-8197-4abe-8ee7-b536f18d0665","name":"Yaali","owner":"abdi","description":"polygon mapping on the field","private":false,"is_public":true,"created_at":"2022-06-05T18:57:51.409650+02:00","updated_at":"2022-06-05T18:57:51.409687+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"failed","user_role":"reader","user_role_origin":"public"},{"id":"1ba2cce1-6a96-4119-863e-3818263f2d6e","name":"percobaan","owner":"abdill…

In web and QGIS everything is fine but on QField I can't sync project (getting 3 dots on bottom)

**Reply by Alekssssanderr** [reply]
![image](https://user-images.githubusercontent.com/110682360/183106089-c3a14750-247a-410c-add5-f6fa902ae70f.png)

**Reply by Notramus** [reply]
That's funny! For me it is the other way around. How long do you have that problem?

**Reply by Alekssssanderr** [reply]
For about 1,5 h. Problem on every mobile account of my collaborators. In qfield log there is information about bad connection.


[HTTP/200] https://app.qfield.cloud/api/v1/projects/?include-public=true Błąd sieci.[{"id":"c0f8877b-8197-4abe-8ee7-b536f18d0665","name":"Yaali","owner":"abdi","description":"polygon mapping on the field","private":false,"is_public":true,"created_at":"2022-06-05T18:57:51.409650+02:00
[{"id":"c0f8877b-8197-4abe-8ee7-b536f18d0665","name":"Yaali","owner":"abdi","description":"polygon mapping on the field","private":false,"is_public":true,"created_at":"2022-06-05T18:57:51.409650+02:00","updated_at":"2022-06-05T18:57:51.409687+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"failed","user_role":"reader","user_role_origin":"public"},{"id":"1ba2cce1-6a96-4119-863e-3818263f2d6e","name":"percobaan","owner":"abdillah","description":"","private":false,"is_public":true,"created_at":"2022-06-27T03:30:23.383221+02:00","updated_at":"2022-08-04T18:09:20.432704+02:00","data_last_packaged_at":"2022-08-04T18:09:07.369279+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"0f50af1f-e7ef-40ba-a893-273a6d7f87fb","name":"FTTH","owner":"abdulrahman","description":"Abd","private":false,"is_public":true,"created_at":"2022-04-24T22:36:16.890837+02:00","updated_at":"2022-04-24T22:36:16.890905+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"failed","user_role":"reader","user_role_origin":"public"},{"id":"173ac4f5-f9e1-4645-a7cc-bdb6a8c6f02f","name":"ResourceMappinng","owner":"abebe","description":"Natural Resource and Mobility Route Mapping in Pastoral Areas of Ethiopia","private":false,"is_public":true,"created_at":"2021-12-19T23:08:57.150848+01:00","updated_at":"2022-08-04T14:01:55.435230+02:00","data_last_packaged_at":"2022-08-04T14:00:04.880618+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"d8a944e6-4a94-43a4-8344-31f594083945","name":"75itapibge","owner":"abimael","description":"Mapeamento","private":false,"is_public":true,"created_at":"2022-06-10T15:12:24.802033+02:00","updated_at":"2022-08-04T14:04:07.980339+02:00","data_last_packaged_at":"2022-08-04T14:03:58.552313+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"dd488f9c-f3a4-4c75-9e0d-a820a979a75f","name":"Albert","owner":"abn","description":"","private":false,"is_public":true,"created_at":"2022-05-23T12:15:27.097573+02:00","updated_at":"2022-07-13T17:08:51.064693+02:00","data_last_packaged_at":"2022-07-13T17:08:41.252810+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"f0490fc7-6c1b-4a89-9d79-a7cb3501f610","name":"RS8","owner":"abubakr","description":"Geol.","private":false,"is_public":true,"created_at":"2022-05-09T16:10:36.605775+02:00","updated_at":"2022-05-09T16:10:36.605817+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"failed","user_role":"reader","user_role_origin":"public"},{"id":"7b2c0499-4535-4a8c-8c31-ca4b7ae80508","name":"webmap","owner":"accelerator","description":"","private":false,"is_public":true,"created_at":"2022-06-10T18:32:12.493915+02:00","updated_at":"2022-07-30T15:20:39.749934+02:00","data_last_packaged_at":"2022-07-30T15:20:32.415254+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"e9075172-d4f1-44f9-a673-936d96d5f202","name":"Adib83","owner":"adib83","description":"Lengkuas","private":false,"is_public":true,"created_at":"2022-05-10T07:43:34.760272+02:00","updated_at":"2022-08-02T21:04:47.116136+02:00","data_last_packaged_at":"2022-08-02T21:04:38.191782+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"87dded0c-384a-4b88-8b5e-d144d0e5fd86","name":"Cholestan","owner":"adnan","description":"","private":false,"is_public":true,"created_at":"2022-06-23T19:36:27.623512+02:00","updated_at":"2022-07-20T17:03:48.133102+02:00","data_last_packaged_at":"2022-07-20T17:02:26.125429+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"fb0379ab-4a43-41c2-b943-6a76fddd40af","name":"Cholestan12345","owner":"adnan","description":"","private":false,"is_public":true,"created_at":"2022-06-23T19:37:55.858761+02:00","updated_at":"2022-06-23T19:38:53.713160+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"585c03f3-6b91-4c38-b7e6-d42067ba1623","name":"NationalGrid","owner":"adriancruz20","description":"","private":false,"is_public":true,"created_at":"2022-06-10T09:55:02.275945+02:00","updated_at":"2022-06-29T15:08:22.202954+02:00","data_last_packaged_at":"2022-06-29T15:08:13.945485+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"2c263743-86e9-476b-8021-2fc38feddb6b","name":"natlgrid","owner":"adriancruz20","description":"","private":false,"is_public":true,"created_at":"2022-06-10T09:55:33.837815+02:00","updated_at":"2022-06-10T09:56:33.396392+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"14e9e467-43fb-448c-8c73-b57aa1c78c1d","name":"NatlGrid","owner":"adriancruz20","description":"","private":false,"is_public":true,"created_at":"2022-06-10T09:55:26.188548+02:00","updated_at":"2022-07-17T08:40:41.459138+02:00","data_last_packaged_at":"2022-07-17T08:40:33.528340+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"0ba2f689-fb04-4d9e-a7ea-d01309ce007c","name":"Boomvalk_Den_Haag_2022","owner":"adri_clements","description":"","private":false,"is_public":true,"created_at":"2022-04-06T08:52:51.659859+02:00","updated_at":"2022-07-24T03:30:39.606020+02:00","data_last_packaged_at":"2022-07-24T03:30:25.570986+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"8451030d-27fb-4b4c-a3ad-e9850ca2f0f7","name":"Sachin","owner":"adwallz","description":"","private":false,"is_public":true,"created_at":"2022-06-29T07:58:56.530818+02:00","updated_at":"2022-07-10T10:21:03.854951+02:00","data_last_packaged_at":"2022-07-10T10:20:50.138465+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"c2171319-542d-4c1d-9d51-4fd177ff237a","name":"erdo","owner":"aersek66","description":"","private":false,"is_public":true,"created_at":"2022-05-19T16:59:57.591322+02:00","updated_at":"2022-07-09T22:00:59.991354+02:00","data_last_packaged_at":"2022-07-09T22:00:50.331156+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"55cfe650-7655-41cc-94c8-9c347e374be3","name":"PARTEI-Test","owner":"agaiagaiagai","description":"","private":false,"is_public":true,"created_at":"2022-03-21T01:22:52.598770+01:00","updated_at":"2022-06-24T19:44:29.977511+02:00","data_last_packaged_at":"2022-06-24T19:44:21.393901+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"a06ca82a-07ac-4114-a993-9244f8297c58","name":"Canalisation","owner":"agent","description":"AEP_ASS","private":false,"is_public":true,"created_at":"2022-06-30T08:54:28.729093+02:00","updated_at":"2022-06-30T08:54:28.729204+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"failed","user_role":"reader","user_role_origin":"public"},{"id":"899c4933-3cce-45d6-a155-3fb7231eba8b","name":"Reseaux","owner":"agent","description":"AEP_ASS","private":false,"is_public":true,"created_at":"2022-06-30T08:41:39.659641+02:00","updated_at":"2022-07-10T17:52:46.840433+02:00","data_last_packaged_at":"2022-07-10T17:52:38.653376+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"20b1c787-5b9d-49e5-b5d9-f5c03de39db2","name":"Telepulesfelmeresbeadando","owner":"agotaalacs","description":"FAGY beadando","private":false,"is_public":true,"created_at":"2022-05-10T20:34:41.328376+02:00","updated_at":"2022-07-13T09:21:21.744123+02:00","data_last_packaged_at":"2022-07-13T09:21:06.052887+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"ed6d233d-931b-46e7-b1c1-80273361cc9b","name":"Minor_Water_Source","owner":"Agricultural_Engineering_Department","description":"Mapping of small water sources like farm pond, Small ponds, Kulam, Oorani etc","private":false,"is_public":true,"created_at":"2022-05-16T05:00:31.352032+02:00","updated_at":"2022-06-26T18:03:57.189912+02:00","data_last_packaged_at":"2022-06-26T18:03:48.134693+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"a3b70196-cb7a-413b-a48a-c700726b532f","name":"Ahmad_Candra_1","owner":"ahmad","description":"","private":false,"is_public":true,"created_at":"2022-06-13T06:32:39.222561+02:00","updated_at":"2022-06-29T05:00:57.128211+02:00","data_last_packaged_at":"2022-06-29T05:00:18.372025+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"4d26d5f1-ff10-4489-a28f-e756a36742d4","name":"kikwit","owner":"ahmedaissaoui1993","description":"","private":false,"is_public":true,"created_at":"2022-03-30T11:14:21.187714+02:00","updated_at":"2022-07-15T17:23:06.363464+02:00","data_last_packaged_at":"2022-07-15T17:22:53.307401+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"ee83dd3d-9ac8-439e-b809-75e532a93c58","name":"test","owner":"ahmederraji","description":"test","private":false,"is_public":true,"created_at":"2022-03-07T10:25:25.943062+01:00","updated_at":"2022-07-27T10:45:31.446554+02:00","data_last_packaged_at":"2022-07-27T10:45:20.502414+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"64c56bf2-064b-4d95-a989-6779cf7e37f2","name":"prueba1","owner":"albinstein","description":"","private":false,"is_public":true,"created_at":"2022-08-04T07:18:09.279954+02:00","updated_at":"2022-08-04T07:18:24.569058+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"6235f01f-6ef6-488b-83b7-ab35cc1b2c70","name":"INSPECCIONES","owner":"ale87arg","description":"datos base para inspeccion en campo de la SsRH","private":false,"is_public":true,"created_at":"2022-04-04T17:10:59.360254+02:00","updated_at":"2022-06-21T11:10:16.069208+02:00","data_last_packaged_at":"2022-06-21T11:10:06.503151+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"45bbfd41-1059-4063-93c3-d19e5ced594e","name":"PNSPM","owner":"alejandrogg","description":"Usos de suelo Monitoreo","private":false,"is_public":true,"created_at":"2022-05-12T00:03:26.350776+02:00","updated_at":"2022-05-12T00:03:36.174787+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"242fd206-1080-4d80-a07a-017c9e4106db","name":"Mat","owner":"alejomoncada","description":"Prueba","private":false,"is_public":true,"created_at":"2022-05-04T21:43:28.285116+02:00","updated_at":"2022-06-27T02:25:00.758024+02:00","data_last_packaged_at":"2022-06-27T02:24:52.303561+02:00","data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"5ad678fd-687e-4855-b2e1-f21431d353bb","name":"test_with_two_layers","owner":"aleksandar_cvejovic","description":"try","private":false,"is_public":true,"created_at":"2022-08-02T10:15:09.686414+02:00","updated_at":"2022-08-02T10:15:37.076250+02:00","data_last_packaged_at":null,"data_last_updated_at":null,"can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"reader","user_role_origin":"public"},{"id":"3f3ad075-1746-478e-8403-0ab0a1ddc163","name":"Bogdan_Borylo_082022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-08-04T09:55:08.546466+02:00","updated_at":"2022-08-04T14:39:44.295907+02:00","data_last_packaged_at":"2022-08-04T14:39:33.233566+02:00","data_last_updated_at":"2022-08-04T14:07:13.396648+02:00","can_repackage":true,"needs_repackaging":false,"status":"ok","user_role":"admin","user_role_origin":"project_owner"},{"id":"c9200cd0-5dc6-4ff2-b0dc-055a74c42e15","name":"Bozena_Rafal_Petlicki_082022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-08-04T18:54:01.212015+02:00","updated_at":"2022-08-04T18:54:23.394958+02:00","data_last_packaged_at":null,"data_last_updated_at":"2022-08-04T18:54:13.527384+02:00","can_repackage":true,"needs_repackaging":true,"status":"ok","user_role":"admin","user_role_origin":"project_owner"},{"id":"70e32065-9352-46ce-803e-6456647c6075","name":"Cieszynskie_zaklady_kartoniarskie_072022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-07-24T10:22:13.542678+02:00","updated_at":"2022-08-05T14:40:54.216968+02:00","data_last_packaged_at":"2022-08-05T14:40:36.586091+02:00","data_last_updated_at":"2022-08-05T14:38:55.336488+02:00","can_repackage":true,"needs_repackaging":false,"status":"ok","user_role":"admin","user_role_origin":"project_owner"},{"id":"6a673231-42d2-4634-abcd-7fa55e412ddb","name":"Edward_Grabowski_072022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-07-28T19:34:51.815382+02:00","updated_at":"2022-07-28T20:19:06.099273+02:00","data_last_packaged_at":"2022-07-28T20:18:51.948604+02:00","data_last_updated_at":"2022-07-28T20:18:38.723535+02:00","can_repackage":true,"needs_repackaging":false,"status":"ok","user_role":"admin","user_role_origin":"project_owner"},{"id":"afd8b0b8-b70f-4316-9a63-2e2f5d964a16","name":"Gospodarstwo_Ruszkowo_072022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-07-28T20:37:35.523534+02:00","updated_at":"2022-07-28T20:43:26.552027+02:00","data_last_packaged_at":"2022-07-28T20:43:14.412319+02:00","data_last_updated_at":"2022-07-28T20:42:21.897661+02:00","can_repackage":true,"needs_repackaging":false,"status":"ok","user_role":"admin","user_role_origin":"project_owner"},{"id":"596016a0-ab58-428b-bea0-1d6cad2f337f","name":"GR_Czyszek_072022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-07-13T12:56:08.457899+02:00","updated_at":"2022-07-13T13:19:14.105946+02:00","data_last_packaged_at":"2022-07-13T13:18:39.787889+02:00","data_last_updated_at":"2022-07-13T12:56:18.676420+02:00","can_repackage":true,"needs_repackaging":false,"status":"ok","user_role":"admin","user_role_origin":"project_owner"},{"id":"922da3aa-eebe-40ea-9f38-748507f16a96","name":"Izabela_Galka_Anna_Krakowiak_072022","owner":"alekssssander","description":"","private":true,"is_public":false,"created_at":"2022-07-31T21:56:21.962601+02:00","updated_at":"2022-08-04T10:15:17.132793+02:00","data_l

**Comment by Alekssssanderr** [comment]
Is there any chance that broken public project doing this failed sync? Can I shut off syncing public projects?

**Comment by Notramus** [comment]
I have honestly no idea. Gotta look into it later tomorrow... if you have
any solution or new information.Please let me know

**Comment by Bvpraagh** [comment]
Sorry I don’t have an answer but same thing has happened to my colleague and myself. All     Cloud projects that have not been locally down loaded are missing from project overview in qfield App. All community projects disappeared as well. All are present on Qfield login on computer.

**Reply by robinjonesuk** [reply]
I have the same problem after a failed synchronization a couple of days ago. No private or public projects showing. I've also found that if I create a new project using the website it results in error 500, but only if I choose "Create a project with a .qgs file with OpenStreetMap as a base layer". Could these be connected?

**Comment by oilahk** [comment]
Also got the same problem

**Comment by suricactus** [comment]
@Bvpraagh @oilahk @robinjonesuk @Notramus @Alekssssanderr looking into this now.

Is it like you are experiencing this error all the time, or sometimes it works for you?

EDIT Ok, it's a known issue that exploded just on extended weekend. It is addressed with the next version.

**Reply by Alekssssanderr** [reply]
All the time :) I tried on different devices and android emulator with some VPN and didn't work at all. On WEB cloud I can see every project. For me it's only qgis and qfield problem with syncing. Hope you solve that easy! Thanks!

**Reply by suricactus** [reply]
Can you try now? Projects should load a bit slow, but now load for me.

**Reply by Alekssssanderr** [reply]
yes, it's syncing now !

**Reply by suricactus** [reply]
Ok, it's a known issue that exploded just on extended weekend. It is addressed with the next version.

**Reply by oilahk** [reply]
@suricactus Thanks! Its working now...

**Reply by Alekssssanderr** [reply]
I was check uploudnig changes on qfield and it's too working! Thanks You a lot! :)

**Comment by Alekssssanderr** [comment]
I tested now pushing changes again and I can push to cloud but changes
after push didn't show on qfield :/

niedz., 7 sie 2022 o 12:30 oilahk ***@***.***> napisał(a):

**Reply by Alekssssanderr** [reply]
![image](https://user-images.githubusercontent.com/110682360/183287830-c678ae89-8d27-4a53-b237-d2e08c4d55f9.png)
points collected on Qfield and pushed to cloud shows on QGIS after sync but on QField disappear

**Reply by robinjonesuk** [reply]
Projects are now showing and syncing again and create project no longer gives error 500. 
Thanks!

**Reply by Azik86rus** [reply]
So when syncing, the points that were entered earlier are not displayed

**Reply by Alekssssanderr** [reply]
@Azik86rus yes. They are not displayed only on Qfield. On QGIS after sync and download latest changes from cloud they are displayed.

**Reply by Alekssssanderr** [reply]
@suricactus Can you look up again at this?

**Reply by suricactus** [reply]
Please open a new issue, with your username and project of interest.

**Comment by Azik86rus** [comment]
There are no points in the mobile version, even if you do synchronization

вс, 7 авг. 2022 г. в 17:37, Alekssssanderr ***@***.***>:

**Reply by suricactus** [reply]
Please open a new issue, with your username and project of interest.
